pkg/nfd/vars.go

```diff
@@ -0,0 +1,21 @@
+package nfd
+
+import . "github.com/rh-ecosystem-edge/nvidia-ci/pkg/global"
+
```

Comment:
Looks like `config.go` is a better name for this file. It can also have functions for reading/calculating the NFD test configuration, such as injecting default values, figuring values based on other values (e.g. if a fallback catalog image is specified, then deploy custom NFD catalog is `true`).